OpenSim/Common/Component.h

```diff
@@ -2793,9 +2793,7 @@ void Input<T>::connect(const AbstractOutput& output,
             else {
                 pathName = pathName + "/" + chan.second.getName();
             }
-            if (!annotation.empty() && annotation != chan.second.getChannelName()) {
-                pathName += "(" + annotation + ")";
-            }
+            pathName += "(" + alias + ")";
```

Comment:
If we do switch to `()` then it seems that the documentation (for parseConnecteeeName()) and (new) tests may need to be updated to reflect this? Did I miss this in the documentation/tests?